selfdrive/car/toyota/carstate.py

```diff
@@ -128,6 +128,19 @@ def update(self, cp, cp_cam):
       ret.leftBlindspot = (cp.vl["BSM"]["L_ADJACENT"] == 1) or (cp.vl["BSM"]["L_APPROACHING"] == 1)
       ret.rightBlindspot = (cp.vl["BSM"]["R_ADJACENT"] == 1) or (cp.vl["BSM"]["R_APPROACHING"] == 1)
 
+    if self.CP.carFingerprint != CAR.PRIUS_V:
```

Comment:
We should just copy the whole message

Reply:
For the Prius V? This message doesn't get sent by the camera (or exist at all) on the V so I don't think you can copy it.

As for the other cars, openpilot still uses this message for other stuff like LDA and cancel chime masking.

Reply:
No I mean create self.lkas_hud and copy the whole dictionary, like the other makes do